rct229/ruleset_functions/get_baseline_system_types.py

```diff
@@ -0,0 +1,4 @@
+def mock_get_baseline_system_types(rmd):
```

Comment:
As I suggested above, I think we can do without a mock function.
That would allow us more flexibilty while we wait for the actual implementation of _get_baseline_system_types_.